src/main/java/com/groupseven/sysc4806project/Book.java

```diff
@@ -0,0 +1,85 @@
+package com.groupseven.sysc4806project;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+import java.awt.*;
+import java.io.Serializable;
+@Entity
+public class Book implements Serializable {
```

Comment:
Does this class need to be serializable? I don't think we'll be serializing this apart from turning it to JSON in our RestControllers, but that's done automatically.

Reply:
I'm not quite sure, but since in the lab, I have something similar so I just put it there. We could test and find out

Reply:
I don't remember using Serializable in my labs. Can you test with/without before we merge?

Reply:
I didnt use serializable either, don't think it was necessary for the labs

Reply:
You guys are right, no need for serializable. It still work